app/src/main/java/com/github/sdp/mediato/model/Review.java

```diff
@@ -60,4 +63,13 @@ public void setComment(String comment) {
         this.comment = comment;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if (!(obj instanceof Review)) {
+            throw new IllegalArgumentException("Object is not an instance of Review");
+        }
+        Review other = (Review) obj;
+        return Objects.equals(this.username, other.username) && Objects.equals(this.media, other.media) && this.grade == other.grade && Objects.equals(this.comment, other.comment);

```

Comment:
Is there a reason why `Objects.equals(this.x, other.x)` is used instead of `this.x.equals(other.x)` ? If I am not wrong, the Media class has two functions (`equals` and `isTheSame`), that allows to compare two `Medias`.